client/state/notices/middleware.js

```diff
@@ -34,6 +42,8 @@ export function dispatchError( message ) {
 	return ( dispatch ) => dispatch( errorNotice( message ) );
 }
 
+export const dispatchErrorMessage = ( dispatch, { error } ) => dispatch( errorNotice( error.message ) );
```

Comment:
That the message is furnished by the error object and not passed into the function. It allows us to display the error message that came back from the server.